src/repror/internals/conf.py

```diff
@@ -84,16 +90,18 @@ def load_recipe_config_and_path(self, clone_dir: Optional[Path] = None) -> dict:
             return conf, path
 
     def load_local_recipe_config(
-        self, recipe_path: Optional[str] = None
+        self, recipe_path: Optional[Path] = None
     ) -> Tuple[dict, Path]:
-        path = Path(recipe_path) if recipe_path else Path(self.local_path)
+        path = recipe_path if recipe_path else Path(self.local_path)
         with path.open("r", encoding="utf8") as file:
             conf = yaml.safe_load(file)
             return conf, path
 
     def load_remote_recipe_config(
         self, clone_dir: Optional[Path] = None
     ) -> Tuple[dict, Path]:
+        if self.url is None:
```

Comment:
I'm more opinnionated to use 
if not `self.url`  for two reasons:
1) we check for None
2) we check if url is not empty

wdyt?

Reply:
Yes, Ill change it.